packages/utils/testing/src/lib/events.js

```diff
@@ -40,7 +40,18 @@ class Events {
 
   request2(cmd, ...args) {
     assert(this.commandHandlers[cmd], `command handler for ${cmd} not registered`);
-    this.commandHandlers[cmd](...args);
+    return new Promise((resolve, reject) => {
+      args.push((err, ...res) => {
+        if (err) {
+          return reject(err);
+        }
+        if (res.length && res.length > 1) {
+          return resolve(res);
+        }
+        return resolve(res[0]);
```

Comment:
Would this be better?

```
if (!res.length) return resolve();
resolve(res[0]);
```

Reply:
I can add that change. Just to be sure, this is not how the actual `request2` API behaves like at the moment. It seems it just expects that, when this API is used, the event *has* to return something. 

But I agree that it should be possible to await an event and not expect an actual value while still resolving 👍

We should probably update our core API as well then.

Reply:
Agreed, it should be consistent.